tests/test_test_utils.py

```diff
@@ -58,7 +69,39 @@ def test_get_route():
                 resp = yield from client.request("GET", "/")
                 assert resp.status == 200
                 text = yield from resp.text()
-                assert "Hello, world" in text
+                assert _hello_world_str == text
+
+            loop.run_until_complete(test_get_route())
+
+
+def test_auto_gzip_decompress():
+    with loop_context() as loop:
+        app = _create_example_app()
+        with _TestClient(app, loop=loop) as client:
```

Comment:
The fixture `test_client` won't work... I just realized that this fixture is overridden in the test file itself.